bark/world/evaluation/rss/rss_interface.hpp

```diff
@@ -90,32 +90,45 @@ typedef std::unordered_map<objects::AgentId, std::pair<bool, bool>>
 class RssInterface {
  public:
   RssInterface() {}
-  explicit RssInterface(const std::string& opendrive_file_name,
-                        const double acc_lon_max, const double brake_lon_max,
-                        const double brake_lon_min,
-                        const double brake_lon_min_correct,
-                        const double acc_lat_brake_max,
-                        const double acc_lat_brake_min,
-                        const double fluct_margin, const double time_response,
-                        const double scaling_relevant_range,
-                        const double route_predict_range)
-      : acc_lon_max_(acc_lon_max),
-        brake_lon_max_(brake_lon_max),
-        brake_lon_min_(brake_lon_min),
-        brake_lon_min_correct_(brake_lon_min_correct),
-        acc_lat_brake_max_(acc_lat_brake_max),
-        acc_lat_brake_min_(acc_lat_brake_min),
-        fluct_margin_(fluct_margin),
-        time_response_(time_response),
-        scaling_relevant_range_(scaling_relevant_range),
-        route_predict_range_(route_predict_range) {
+  explicit RssInterface(std::string opendrive_file_name,
+                        const commons::ParamsPtr& params) {
+    acc_lon_max_ = params->GetReal(
+      "EvaluatorRss::AccLonMax", "maximum acceleration", 1.7);
+    brake_lon_max_ = params->GetReal(
+      "EvaluatorRss::BrakeLonMax", "maximum deceleration", -1.7);
+    brake_lon_min_ = params->GetReal(
+      "EvaluatorRss::BrakeLonMin", "minimum braking deceleration", -1.69);
+    brake_lon_min_correct_ = params->GetReal(
+      "EvaluatorRss::BrakeLonMinCorrect",
+      "minimum deceleration of oncoming vehicle", -1.67);
+    acc_lat_brake_max_ = params->GetReal(
```

Comment:
I would prefer that we have to completely different sets of parameters for ego and others. Not only the response time. Since there may be request in the future for instance, to have different acc. limits for ego and others.

Reply:
Was thinking about that as well!
My suggestion would be to introduce a struct class and then have two of these as members containing all parameters